zdm/survey.py

```diff
@@ -852,6 +973,9 @@ def process_survey_file(self,filename:str,
             self.meta['WIDTH'] = np.median(self.frbs['WIDTH'])
             self.meta['DMG'] = np.mean(self.frbs['DMG'])
         
+        # fills in missing coordinates is possible
```

Comment:
I should move this to before the cuts section, since fixing coordinates to get correct cut values for Gb (i.e. in RA and DEC are set)

Reply:
OK, I have done this now